content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+future vulnerabilities too.
+
+[KEP-vulns]: https://github.com/kubernetes/enhancements/tree/217d790720c5aef09b8bd4d6ca96284a0affe6c2/keps/sig-node/127-user-namespaces#motivation
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Furthermore, you also need support in your container runtime to use this feature
+with Kubernetes stateless pods:
+
+* CRI-O: v1.25 has support for user namespaces.
+
+* containerd: support is planned for the 1.7 release. See containerd
+  issue [#7063][containerd-userns-issue] for more details.
+
+Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
+
+Please note that if your container runtime doesn't support user namespaces, the
+new `pod.spec` field will be silently ignored and the pod will be created without
+user namespaces.
+
+[CRI-dockerd-issue]: https://github.com/Mirantis/cri-dockerd/issues/74
+[containerd-userns-issue]: https://github.com/containerd/containerd/issues/7063
+
+<!-- steps -->
+
+## Configure a Pod
+
+A user namespace for a stateless pod is enabled setting the `hostUsers` field of
+`.spec` to false. For example:
+
+{{< codenew file="pods/user-namespaces-stateless.yaml" >}}
+
+1. Create the pod on your cluster:
+
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/user-namespaces-stateless.yaml
+   ```
+
+1. Attach to the container and run `cat /proc/self/uid_map`:
+
+   ```shell
+   kubectl attach -it userns bash
+   ```
+
+And run the command. The output is similar to this:
+
+```none
+root@userns:/# cat /proc/self/uid_map
+         0     131072      65536
+```
+
+This means that UID 0 is mapped to UID 131072 in the host and this is done for
+65536 consecutive UIDs. In other words, UID 1 in the container is mapped to UID
+131072+1 on the host, and so on.
+
+For more details, please see `man 7 user_namespaces`.
+
+## Verify if user namespaces is active for a pod
+
+To verify that a pod is using user namespaces, if you exec into the pod you can
+run:
+
+```none
+root@userns:/# cat /proc/self/uid_map
+         0     131072      65536
+root@userns:/# cat /proc/self/gid_map
+         0     131072      65536
+```
+
+User namespaces are active if the last value is 65536 in both cases. If you see
+that the last value is 4294967295, this means user namespaces are not active.
+
+The value 4294967295 is the max number of UIDs/GIDs a Linux system can use. If
+you see the whole range of UIDs/GIDs available to the pod, it means a user
+namespaces is not being used.
+
+If your pod's containers doesn't have a shell, you can do it from the host.
+
+If you know which PID in the host the container process is using, you can read
+from the host the same files: `cat /proc/$pid/uid_map` and `cat
+/proc/$pid/gid_map`, replacing `$pid` with the container PID.
+
+There are others ways that can be used to check if user namespaces are active or
+not. This section is not intended to list them all.
```

Comment:
I'd omit this text; it's not that helpful for a task page.

Reply:
I removed that and what to do if the pod doesn't have a shell, as our example does have a shell. I think it will be nice to give those tips to users, but now that everything is separated in concept and example pages, not sure this fits. Let me know if you think it is better to add it back :)